### PR TITLE
WIP Update base services tests to SDK style projects

### PR DESCRIPTION
**Pull request by chsienki**
Update all the tests in baseservices to be SDK style projects

### CODE REVIEW

**Comment by chsienki** [comment]
@dotnet-bot test Windows_NT x86 Checked Build and Test
@dotnet-bot test Windows_NT x64 Build and Test

File: tests/src/baseservices/compilerservices/RuntimeWrappedException/RuntimeWrappedException.csproj [hunk 1]
```diff
@@ -1,40 +1,12 @@
...
-  </ItemGroup>
-  <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
-  </ItemGroup>
-  <ItemGroup>
-      <ProjectReference Include="StringThrower.ilproj" />
-  </ItemGroup>
-  <ItemGroup>
+    <ProjectReference Include="StringThrower.ilproj" />
     <NoWarn Include="42016,42020,42025,42024" />
```
Comment (4creators) [inline tests/src/baseservices/compilerservices/RuntimeWrappedException/RuntimeWrappedException.csproj]: That's wrong semantics as all upper level NoWarn settings are lost - should be 
```xml
<NoWarn Include="42016,42020,42025,42024,$(NoWarn)" />
```

I know that it is old code but perhaps you could fix it with some freecycles, or not 😄

File: tests/src/baseservices/compilerservices/dynamicobjectproperties/Dev10_535767.csproj [hunk 2]
```diff
@@ -1,43 +1,15 @@
...
-  </ItemGroup>
   <ItemGroup>
     <!-- Add Compile Object Here -->
     <Compile Include="dev10_535767.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
-  </ItemGroup>
-  <ItemGroup>
     <ProjectReference Include="../../../Common/CoreCLRTestLibrary/CoreCLRTestLibrary.csproj" />
```
Comment (4creators) [inline tests/src/baseservices/compilerservices/dynamicobjectproperties/Dev10_535767.csproj]: All absolute references to CoreCLRTestLibrary.csproj were causing a lot of problems - mainly warnings. I have fixed to them to relative paths rooted in `tests\src\` directory - pls see #19275 commit d265f21fdcbba61d8421b6941c2fb078ae9fc06d

File: tests/src/baseservices/compilerservices/dynamicobjectproperties/TestAPIs.csproj [hunk 3]
```diff
@@ -1,43 +1,15 @@
...
   <ItemGroup>
     <!-- Add Compile Object Here -->
     <Compile Include="helper.cs" />
     <Compile Include="testapis.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
-  </ItemGroup>
-  <ItemGroup>
     <ProjectReference Include="../../../Common/CoreCLRTestLibrary/CoreCLRTestLibrary.csproj" />
```
Comment (4creators) [inline tests/src/baseservices/compilerservices/dynamicobjectproperties/TestAPIs.csproj]: same as above

**Comment by 4creators** [comment]
@chsienki Pls take a look at #19275 as the changes are partially overlapping and conflicting

**Comment by ViktorHofer** [comment]
Hey @chsienki, what's the status on this? I would really love to see this going into master.

**Comment by jkotas** [comment]
Closing stale PRs.

**Pull request closed without merging**